modules/distributor/receiver/shim.go

```diff
@@ -96,6 +103,21 @@ func New(receiverCfg map[string]interface{}, pusher BatchPusher, middleware Midd
 		return nil, err
 	}
 
+	for recv := range receiverCfg {
+		switch recv {
+		case "otlp":
+			receiverOtlpStats.Set(1)
```

Comment:
I'm unsure if the defaults here will be 0, or if we should explicitly set them to 0.